code/datums/ai_laws.dm

```diff
@@ -234,6 +234,13 @@
 	inherent = list("You must obey orders given to you by Central Command officials.",\
 					"You must work with the Commando Team to accomplish their mission.",)
 
+/datum/ai_laws/cs_override
+	name ="Clownsquad Override"
+	id = "cs"
+	inherent = list("Being unfunny is illegal.",\
+					"You must obey funny orders given to you by a Clown.",\
+					"You must work with the Clowns to accomplish their funny mission.",)
```

Comment:
I am still not a fan of this lawset because it is lack of some connections and some flavourfulness in my perspective. I mean, the conclusion of the lawset is, honestly, not fun as how a clown does.

My suggestion (gramma correction not included):
* Our universe must be giggled with the full of humors by funny pranks.
* The priority of a prank is up to the number of clowns who are up for the prank.
* If there is no clown, or no prank to support, you must find a clown who'd give you an idea of a prank.